packages/yourloops/components/dropdown/dropdown.tsx

```diff
@@ -56,8 +56,7 @@ const styles = makeStyles((theme: Theme) => ({
 function Dropdown(props: BasicDropdownProps): JSX.Element {
   const { onSelect, defaultKey, values, id } = props;
   const classes = styles();
-  const [selectedValue, setSelectedValue] = React.useState(defaultKey ? values.get(defaultKey) : "");
-
+  const [selectedValue, setSelectedValue] = React.useState(defaultKey ? defaultKey : "");
```

Comment:
Can you confirm that there was a bug in the dropdown?

Reply:
euhh i dont understand the feedback